src/dialog/savedqueries/dlgsavedquerieseditor.cpp

```diff
@@ -0,0 +1,50 @@
+#include <QSqlTableModel>
+
+#include "dialog/savedqueries/dlgsavedquerieseditor.h"
+#include "dialog/savedqueries/savedqueriestablemodel.h"
+#include "library/libraryfeature.h"
+#include "library/trackcollection.h"
+
+DlgSavedQueriesEditor::DlgSavedQueriesEditor(LibraryFeature* pFeature,
+                                             TrackCollection* pTrackCollection,
+                                             QWidget* parent)
+        : QDialog(parent),
+          m_pTrackCollection(pTrackCollection),
+          m_pFeature(pFeature) {
+    setupUi(this);
+    SavedQueriesTableModel *pSaveModel = 
```

Comment:
I think yes. Who is the owner of pSaveModel? Is is it currently a memory leak? 
In general , we should use  unique_ptr instead of a new and delete pair when possible. 
If the object has a QObject parent we still have raw pointers, that are deleted by the parent.